### PR TITLE
Removed examples for securityType Standard

### PR DESCRIPTION
**Pull request by AjKundnani**
## Description

Goal of implementing Trusted launch as default is to seamlessly enable security for end users.

Sharing examples of security type Standard conflicts with goal of TL-as-default change.

## Mandatory Checklist

- Please choose the target release of Azure PowerShell
  - [x] [General release](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Public preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Private preview](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] [Engineering build](../blob/main/CONTRIBUTING.md#target-release-types)
  - [ ] N/A

- [x] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For SDK-based development mode, update `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`.
        * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. 
    * For autorest-based development mode, include the changelog in the PR description.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/Compute/Compute/help/New-AzVM.md [hunk 1]
```diff
@@ -358,24 +353,6 @@ $vm = Get-AzVM -ResourceGroupName $rgname -Name $vmname;
 ```
 This example shows how to create a VM with a valid Gen2 image, allowing the VM to default to TrustedLaunch which requires Gen2 images. Please check [the Trusted Launch feature page](aka.ms/trustedlaunch) for more information.
 
-### Example 10: Creates a VM with TrustedLaunch turned on by default.
```
Comment (Sandido) [inline src/Compute/Compute/help/New-AzVM.md:361]: This example seems helpful to keep, especially given how invasive the change to TL by Default was.

File: src/Compute/Compute/help/New-AzVM.md [hunk 2]
```diff
@@ -212,7 +210,7 @@ $bytes = [System.Text.Encoding]::Unicode.GetBytes($text);
 $userData = [Convert]::ToBase64String($bytes);
 
 # Create VM
-New-AzVM -ResourceGroupName $ResourceGroupName -Name $VMName -Credential $Credential -DomainNameLabel $domainNameLabel -UserData $userData -SecurityType $securityTypeStnd;
```
Comment (Sandido) [inline src/Compute/Compute/help/New-AzVM.md:215]: Have you walked through all of these examples and run them to verify that they function with Standard removed?

**Review by Sandido — CHANGES_REQUESTED** [review]

File: src/Compute/Compute/help/New-AzVM.md (hunk 1 above)
Comment (AjKundnani) [reply src/Compute/Compute/help/New-AzVM.md:361]: With TL-as-default, our goal is to keep seamless experience for end users. i.e., they're creating VMs using usual cmdlets/scripts without change and now VM is foundationally secure with Trusted launch.

Hence, highlighting TL-as-default change is more invasive will conflict with this goal.

File: src/Compute/Compute/help/New-AzVM.md (hunk 2 above)
Comment (AjKundnani) [reply src/Compute/Compute/help/New-AzVM.md:215]: Validated 3 examples, can confirm validation for all examples tomorrow.

Comment (AjKundnani) [reply src/Compute/Compute/help/New-AzVM.md:215]: @Sandido - Validated all examples.
Removed example 9 from the list as well to ensure TL-as-default is not perceived as invasive.

**Comment by wyunchi-ms** [comment]
Any updates for this PR?@AjKundnani

**Comment by AjKundnani** [comment]
> Any updates for this PR?@AjKundnani

@wyunchi-ms - Resolved inputs and comments shared by @Sandido, notified over email.
Awaiting his review approval.

**Review by Sandido — DISMISSED** [review]

**Comment by Sandido** [comment]
Validated examples all work, changes seem fine.

**Comment by wyunchi-ms** [comment]
Please resolve the conflicts then.

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**